easybot.js

```diff
@@ -0,0 +1,172 @@
+import { driver, settings } from '@rocket.chat/sdk';
+import dotenv from 'dotenv';
+import { config } from './config.js';
+import * as repository from './repository.js';
+dotenv.config();
+settings.username = process.env.USERNAME_ROCKETCHAT ;
+settings.password = process.env.PASSWORD;
+settings.host = process.env.HOST;
+let myuserid;
+
+const runbot = async () => {
+  await repository.createRedisClient();
```

Comment:
if the redis is down/having networking issue the bot will be crashed,
in this service the redis is being used only for performance improvements, 
so this is not a good idea to prevent our functionalities due to unavailable redis container.